Kitodo-LongTimePreservationValidationModule/src/main/java/org/kitodo/longtimepreservationvalidationmodule/KitodoOutputHandler.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.longtimepreservationvalidationmodule;
+
+import edu.harvard.hul.ois.jhove.App;
+import edu.harvard.hul.ois.jhove.HandlerBase;
+import edu.harvard.hul.ois.jhove.Message;
+import edu.harvard.hul.ois.jhove.Module;
+import edu.harvard.hul.ois.jhove.OutputHandler;
+import edu.harvard.hul.ois.jhove.RepInfo;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.UndeclaredThrowableException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import org.kitodo.api.validation.State;
+import org.kitodo.api.validation.ValidationResult;
+
+public class KitodoOutputHandler extends HandlerBase {
+
+    private static final Pattern LINE_SPLITTER = Pattern.compile(System.lineSeparator());
+
+    private static final Constructor<ValidationResult> VALIDATION_RESULT_CONSTRUCTOR;
+
+    static {
+        try {
+            VALIDATION_RESULT_CONSTRUCTOR = ValidationResult.class.getDeclaredConstructor(State.class,
+                Collection.class);
+            VALIDATION_RESULT_CONSTRUCTOR.setAccessible(true);
+        } catch (NoSuchMethodException e) {
+            throw new UndeclaredThrowableException(e);
+        }
+    }
+
+    /**
+     * Method that is executed before the constructor of the parent class.
+     *
+     * @return {@code null}. The value is not used.
+     */
+    private static final String beforeParentConstructor() {
+        Logger.getLogger("edu.harvard.hul.ois.jhove.handler").setLevel(Level.OFF);
+        return null;
+    }
+
+    private final List<String> messages = new ArrayList<String>();
+
+    private ValidationResultState state;
+
+    protected KitodoOutputHandler() {
+        super(beforeParentConstructor(), null, new int[] {1970, 1, 1 }, null, null);
+    }
+
+    private void addMessage(Message message) {
+        StringBuilder line = new StringBuilder(127);
+        line.append(message.getMessage());
+        String subMessage = message.getSubMessage();
+        if (subMessage != null) {
```

Comment:
In other modules an expression like `Objects.nonNull` is used for comparing to not `null`.

Reply:
Javadoc of `Objects.nonNull()`:
> **API Note:**
> This method exists to be used as a [Predicate](https://docs.oracle.com/javase/8/docs/api/java/util/function/Predicate.html), `filter(Objects::nonNull)`

Reply:
But it is not a problem, it can be safely used:
https://stackoverflow.com/questions/37972859/java-util-objects-isnull-vs-object-null

Reply:
Of course it is safe to use, where else would we go? But it is absolutely unnecessary.

The thing is:  
Code reviews should be about getting safe and understandable code. I consider it absolutely not expedient to constantly discuss any functionally identical little things. We win nothing, we lose: time to focus on the essentials. My and yours. So please ...

Reply:
True, but point is here that @henning-gerhardt is right with usage consistent code style in all places. It doesn't hurt to stick to it.

Reply:
It does hurt because of the time it takes to

* switch the IDE to the corresponding branch
* wait for the IDE to compile
* figure out which module is affected, open it
* figure out which file is affected, navigate to it
* find the correct line
* Change it
* find unit tests
* run unit tests
* see that file was not yet saved
* Click save
* run unit tests again
* Find ‘Git Staging’ tab
* Craft a git commit
* Look up some english word in the dictionary that you need for the commit message, but are not sure any more how it is written
* Commit
* open Travis in web browser to see what is happening
* find out that branch was pushed to wrong name, because you renamed it recently when cleaning up, and forgot
* push branch again manually
* … etc …
And two days later:
* find the e-mail sent by GitHub about the comment and spend time to think about if that is something new or something I have already fixed
* your time to check if I changed it. If not: why? Are you on the correct branch? On the latest commit? Check back everything.

To put it simple, *every* remark during code review, and every change is taking time so we should spend it on more valuable development.

By the way, there are 2,130 usages of `someVariable != null` vs. 270 usages of `Objects.nonNull()` in my search, so I cannot see a consistent code style here yet.